src/codehelp/codehelp.vala

```diff
@@ -20,6 +20,272 @@
  * Code help utilities that don't belong to any specific class
  */
 namespace Vls.CodeHelp {
```

Comment:
You should call this `Vls.Formatter`

Reply:
So make a extra file (E.g. formatter.vala) in `codehelp/`?

Reply:
Yes.